src/main/java/com/minecolonies/network/messages/FarmerCropTypeMessage.java

```diff
@@ -85,11 +79,7 @@ public IMessage onMessage(FarmerCropTypeMessage message, MessageContext ctx)
                 BuildingFarmer building = colony.getBuilding(message.buildingId, BuildingFarmer.class);
                 if (building != null)
                 {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00108)